sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -178,6 +178,30 @@ def test_iobase_source(self):
               query=query, use_standard_sql=True, project=self.project))
       assert_that(result, equal_to(self.TABLE_DATA))
 
+  @pytest.mark.it_postcommit
+  def test_table_schema_retrieve(self):
+    the_table = beam.io.gcp.bigquery.bigquery_tools.BigQueryWrapper().get_table(
+        project_id="apache-beam-testing",
+        dataset_id="beam_bigquery_io_test",
+        table_id="dfsqltable_3c7d6fd5_16e0460dfd0")
+    table = the_table.schema
+    utype = beam.io.gcp.bigquery_schema_tools.produce_pcoll_with_schema(table)
+    with beam.Pipeline(argv=self.args) as p:
+      result = (
+          p | beam.io.gcp.bigquery.ReadFromBigQuery(
+              gcs_location="gs://bqio_schema",
+              table="beam_bigquery_io_test.dfsqltable_3c7d6fd5_16e0460dfd0",
+              project="apache-beam-testing")
+          | beam.io.gcp.bigquery.ReadFromBigQuery.get_pcoll_from_schema(table))
```

Comment:
When the previous comment is addressed, we should be able to test this like so:
```suggestion
          p | beam.io.gcp.bigquery.ReadFromBigQuery(
              gcs_location="gs://bqio_schema",
              table="beam_bigquery_io_test.dfsqltable_3c7d6fd5_16e0460dfd0",
              project="apache-beam-testing",
              output_type="BEAM_ROW")
```